bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/RetryListener.java

```diff
@@ -62,10 +64,11 @@ public void onHeaders(Metadata headers) {
   @Override
   public void onClose(Status status, Metadata trailers) {
     if (isRetriableStatus(status.getCode())
-        && isRetriableCall
         && !stateSignalledToListener) {
       if (retryingCall.retryCallAfterBackoff(payload, requestHeaders, this)) {
-        // We are retrying
+        // A retryable error.
+        failedCount += 1;
+        LOG.info("Retrying failed call. Failure #" + failedCount + ", got: " + status, status.getCause());
```

Comment:
Since there is nothing actionable about this logging I would prefer to keep it as info, not warning. One could argue it should be debug...

Seems throwable is the second argument:
https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/master/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/Logger.java#L55